packages/cli-vector/schema/pois.json

```diff
@@ -806,6 +797,20 @@
       "source": "s3://linz-lds-cache/50246/",
       "tags": { "building": "building" },
       "style": { "minZoom": 14, "maxZoom": 15 }
+    },
+    {
+      "id": "50275",
+      "name": "50275-nz-ford-points-topo-150k",
+      "source": "s3://linz-lds-cache/50275/",
+      "tags": { "kind": "secondary", "waterway": "ford" },
```

Comment:
I wonder do we still need the `kind:secondary` here? Can you please check how does the ford been used in style? And is the `kind:secondary` used from the previous `street` layer?
Because, I remember there is no `kind` in the `poi` layer. But if we really need it, we also need add `kind` into `metadata.attributes`.